addons/rookupgrade/10to14/install.sh

```diff
@@ -2,14 +2,18 @@
 
 function rookupgrade_10to14_upgrade() {
     local from_version="$1"
-
     local upgrade_files_path="$DIR/addons/rookupgrade/10to14"
 
     # if it is less than or equal we re-apply in cause of a failure mid upgrade
     if [ "$(rook_upgrade_compare_rook_versions "$from_version" "1.1")" != "1" ]; then
         # this will start the rook toolbox if it doesn't already exist
-        "$DIR"/bin/kurl rook wait-for-health
+        log "Waiting for rook be health"
+        if ! $DIR/bin/kurl rook wait-for-health 300 ; then
+            kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+            bail "Failed to verify the updated cluster, Ceph is not healthy"
+        fi
 
+        log "Updating mon count to actual count"
```

Comment:
```suggestion
        log "Waiting for rook to be healthy"
        if ! $DIR/bin/kurl rook wait-for-health 300 ; then
            kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
            bail "Failed to verify the updated cluster, Ceph is not healthy"
        fi

        log "Updating the Cep mon count"
```

Reply:
There's a few places where 'rook be health' snuck in, and that should be fixed

Reply:
@laverya this PR is just for we start to properly output/get the case scenarios.
But yep, we will need to fix any stuck scenario in follow ups.